src/Slices/ServiceInstanceDetails/Core/Context.tsx

```diff
@@ -0,0 +1,24 @@
+import { createContext } from "react";
+import { UseQueryResult } from "@tanstack/react-query";
+import { ServiceInstanceModel, ServiceModel } from "@/Core";
+import { InstanceLog } from "@/Slices/ServiceInstanceHistory/Core/Domain";
```

Comment:
```suggestion
import { InstanceLog } from "@/Slices/ServiceInstanceHistory/Core/Domain";

```

Reply:
I'm not certain I get what you mean here.

Reply:
sorry, I proposed extra space between import and docstrings :)